lib/newrelic.js

```diff
@@ -67,10 +67,14 @@ class NewRelic {
      * @param {object} defaultMetrics default metrics to include in every New Relic event
      */
     constructor(options, defaultMetrics={}) {
-        this.defaultMetrics = Object.assign(Metrics.openwhisk(), defaultMetrics);
 
+        this.defaultMetrics = Object.assign(Metrics.openwhisk(), defaultMetrics);
         this.canSendMetrics = false;
-        if (!options ||
+
+        if (process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS === 'true') {
```

Comment:
@tmathern Changed to check for `true` as a string since environment variable values are all strings
WDYT?

Reply:
You can check that the value isn't set to false, but I wouldn't check for `true` here. Otherwise we could start checking if it is `enable`, `activate`, `ok`, etc...

Reply:
I'm also thinking that users may go back and forth between setting this to true and false like we do with devtool - we check that an environment variable is set to "development" else ignore it and run with the production setup. Also stated in the documentation is set `OPENWHISK_NEWRELIC_DISABLE_METRICS` to `true` if you want to disable sending metrics. If I read that documentation as a developer, I would think by setting `OPENWHISK_NEWRELIC_DISABLE_METRICS` to `false` would reverse the disabling of metric sending.

On the other hand `OPENWHISK_NEWRELIC_DISABLE_ALL_INSTRUMENTATION` is just being checked if it's set or not using the same documentation wording. 

Just seems like it would cause more trouble if the code only checks if the environment variable contains a value. Also an observation in the wild, some "developers" don't know much about environment variables and tend to set the value to '' thinking that unsets the environment variable.

Reply:
Ok, you convinced me.
PR LGTM, then.

Reply:
(Maybe we should use the same logic for OPENWHISK_NEWRELIC_DISABLE_ALL_INSTRUMENTATION, also. Wdyt @jdelbick ?)

Reply:
@jdelbick ^^

Reply:
New ticket if the check for OPENWHISK_NEWRELIC_DISABLE_ALL_INSTRUMENTATION should change